### PR TITLE
Add the release note for 1.9.2 and update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Micrometer 1.7.0 -> 1.7.1
- ScalaPb 0.11.3 -> 0.11.4
- Spring Boot 2.5.1 -> 2.5.2
- Build
  - finagle-serversets_2.13 21.5.0 -> 21.6.0
  - graphql-dgs-client 4.2.0 -> 4.3.1
  - kotlin-allopen 1.5.10 -> 1.5.20
  - Proguard 7.1.0-beta1 -> 7.1.0

### CODE REVIEW

File: site/src/pages/release-notes/1.9.2.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- N/A
+
+## ☢️ Breaking changes
+
+- N/A
+
+## ⛓ Dependencies
+
+- Micrometer 1.7.0 → 1.7.1
+- ScalaPb 0.11.3 → 0.11.4
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.9.2.mdx:30]:
```suggestion
- ScalaPB 0.11.3 → 0.11.4
```

**Review by ikhoon — APPROVED** [review]
Thanks @minwoox

File: site/src/pages/release-notes/1.9.2.mdx (hunk 1 above)
Comment (minwoox) [reply site/src/pages/release-notes/1.9.2.mdx]: Fixed. 😉

**Review by trustin — APPROVED** [review]
Thanks, @minwoox !

**Pull request merged**